pkg/tagger/collectors/kubelet_extract.go

```diff
@@ -89,6 +89,10 @@ func (c *KubeletCollector) parsePods(pods []*kubelet.Pod) ([]*TagInfo, error) {
 				tags.AddLow("kube_replication_controller", owner.Name)
 			case "StatefulSet":
 				tags.AddLow("kube_stateful_set", owner.Name)
+				persistentvolumeclaim := getPersistentVolumeClaimName(pod)
+				if persistentvolumeclaim != "" {
+					tags.AddLow("persistentvolumeclaim", persistentvolumeclaim)
```

Comment:
we might want to snake_case this, also there can only be one PVC per stateful set?

Reply:
or maybe shorten it to `kube_pvc` ? 🤔

Reply:
The idea is to match the `persistentvolumeclaim ` tag in `kubernetes.kubelet.volume.*` metrics